src/plugins/content/accessibility/jumplinks/shortcuts.js

```diff
@@ -39,14 +71,55 @@ const defaults = {};
  * @returns {shortcutsBox} the component, initialized and rendered
  */
 export default function shortcutsBoxFactory(config) {
+    const shortcutsBox = component({}, defaults)
+        .on('render', function () {
```

Comment:
Why didn't you use the dialog component? Seems it is doing most of the work.

Reply:
I tried to use dialog component, but there was problems with screenreader announcement of the shortcuts, so we decided to stay with it but make it looks like dialog